index.html

```diff
@@ -5,6 +5,7 @@
 <script class='remove' src="respecConfig.js"></script>
 <link rel="stylesheet" href="styles/spec.css">
 
+<body>
```

Comment:
Probably don’t need body.

Reply:
tidy complained about that though

Reply:
Oh, I figured tidy would not work at all with this document as it’s mostly markdown. I thought it would make a mess. That’s awesome if it doesn’t.